docs/source/examples/Notebook/rstversions/Typesetting Equations.rst

```diff
@@ -50,7 +50,7 @@ Source
     \end{equation*}
 
 Display
--------
+~~~~~~~
 
```

Comment:
rst raw directives are ignored if you're producing anything other than that format (like raw notebook cells). We should make a Sphinx rst template that uses the `.. math::` directive for one of the available [math extensions](http://sphinx-doc.org/ext/math.html). But if this is being produced by pandoc, it might be harder to customise...